source/wrapper/test/Wrapper-unit.js

```diff
@@ -329,4 +331,68 @@ contract('Wrapper Unit Tests', async accounts => {
       )
     })
   })
+
+  describe('Test pausability', async () => {
+    it('Test setPausedStatus is only callable by owner', async () => {
+      await reverted(wrapper.setPausedStatus(true, { from: notOwner }))
```

Comment:
maybe check the revert message is `        'Ownable: caller is not the owner'`

Reply:
Interesting. I didn't realize this was added in their contracts.